builtins: implement ST_WKBToSQL and ST_WKTToSQL

otan opened the pull request:
Resolves #48821.
Resolves #48822.

Release note (sql change): Implement the ST_WKBToSQL and ST_WKTToSQL
functions.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/49237)
<!-- Reviewable:end -->

otan committed:
builtins: implement ST_WKBToSQL and ST_WKTToSQL

Release note (sql change): Implement the ST_WKBToSQL and ST_WKTToSQL
functions.

sumeerbhola reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

> Reviewed 3 of 3 files at r1.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/49237#-:-M7szRad0G4DlBNgF8rS:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained



<!-- Sent from Reviewable.io -->

otan commented:
bors r=sumeerbhola
